Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2485,6 +2496,23 @@ extension BrowserViewController: TabManagerDelegate {
 
       bookmarkMenuChildren.append(bookmarkAllTabs)
     }
+    
+    var duplicateTabMenuChildren: [UIAction] = []
+
+    if containsWebPage, let selectedTab = tabManager.selectedTab, let url = selectedTab.fetchedURL {
+      let duplicateActiveTab = UIAction(
+        title: Strings.duplicateActiveTab,
+        image: UIImage(systemName: "plus.square.on.square"),
+        handler: UIAction.deferredActionHandler { _ in
```

Comment:
does this need `[unowned self]` as well?

Reply:
Fixed at https://github.com/brave/brave-ios/pull/5236/commits/0d2fe3d86d839c390e029f0d78408ae0967f5d5d